src/app/lib/utilities/moment/index.stories.jsx

```diff
@@ -18,17 +18,52 @@ import '#server/utilities/serviceConfigs';
 const locales = [
   // some other locales (eg. Russian) have multiple levels of pluralisation (ie. one, some, many) which might require
   // adding more examples to the table below.
+  { name: 'Afaan Oromoo', locale: 'om' },
+  { name: 'Afrique', locale: 'fr' },
+  { name: 'Amharic', locale: 'am' },
+  { name: 'Arabic', locale: 'ar' },
+  { name: 'Azeri', locale: 'az' },
+  { name: 'Bengali', locale: 'bn' },
+  { name: 'Burmese', locale: 'my' },
+  { name: 'Cymrufyw', locale: 'cy' },
+  { name: 'Gahuza', locale: 'rw' },
   { name: 'Gujarati', locale: 'gu' },
+  { name: 'Hausa', locale: 'ha' },
+  { name: 'Hindi', locale: 'hi' },
   { name: 'Igbo', locale: 'ig' },
   { name: 'Indonesian', locale: 'id' },
+  { name: 'Japanese', locale: 'ja' },
   { name: 'Korean', locale: 'ko' },
+  { name: 'Kyrgyz', locale: 'ky' },
   { name: 'Marathi', locale: 'mr' },
+  { name: 'Mundo', locale: 'es' },
+  { name: 'Naidheachdan', locale: 'gd' },
+  { name: 'Nepali', locale: 'ne' },
+  { name: 'News', locale: 'en-gb' },
   { name: 'Pashto', locale: 'ps' },
+  { name: 'Persian', locale: 'fa' },
   { name: 'Pidgin', locale: 'pcm' },
+  { name: 'Portuguese', locale: 'pt-br' },
   { name: 'Punjabi', locale: 'pa-in' },
+  { name: 'Russian', locale: 'ru' },
+  { name: 'Scotland', locale: 'en-gb' },
+  { name: 'Serbian', locale: 'sr' },
+  { name: 'Serbian Cyrillic', locale: 'sr-cyrl' },
+  { name: 'Sinhala', locale: 'si' },
+  { name: 'Somali', locale: 'so' },
+  { name: 'Swahili', locale: 'sw' },
+  { name: 'Tamil', locale: 'ta' },
   { name: 'Telugu', locale: 'te' },
   { name: 'Thai', locale: 'th' },
+  { name: 'Tigrinya', locale: 'ti' },
+  { name: 'Turkce', locale: 'tr' },
+  { name: 'Ukrainian', locale: 'uk' },
+  { name: 'UkChina', locale: 'zh-cn' },
+  { name: 'Urdu', locale: 'ur' },
+  { name: 'Uzbek', locale: 'uz' },
+  { name: 'Vietnamese', locale: 'vi' },
   { name: 'Yoruba', locale: 'yo' },
+  { name: 'Zhongwen', locale: 'yo' },
```

Comment:
I think it would be much nicer to import services from `#server/utilities/serviceConfigs` to make it scalable.  And map over them. If a service is added or `locale` change for some reason we will not need to also update this file.

Reply:
Agree, I also thought about it 👍